utils.py

```diff
@@ -119,7 +144,7 @@ class ModelConfig:
         Update the model configuration dictionary with the current attributes.
     """
 
-    def __init__(self, samplemode, do_fudge, use_disort=0, malk=0, mch4=0, do_bff=1, fresh=0, xpath="../Linelists/", xlist="data/gaslistRox.dat", dist=None, pfile="data/LSR1835_eqpt.dat"):
+    def __init__(self, samplemode, do_fudge, use_disort=0, malk=0, mch4=0, do_bff=1, fresh=0, xpath="../Linelists/", xlist="gaslistRox.dat", dist=None, pfile="LSR1835_eqpt.dat"):
```

Comment:
the new code should point to the data/ directory.   You may need to pull from master in order to test your changes with the newly-organized code.